libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -429,14 +426,69 @@ public boolean checkBirthDateCorrelationInCredentials(String userId)
                 <= 1;
     }
 
+    // This method checks the birthdate requirement, which is in violation of GPG45 rules.
+    // However, considering that BAV and Address CRI  does not contain a birthdate,
+    // we add this special handling only for BAV and Address CRI. For other CRIs, we continue to
+    // validate this
+    // requirement.
```

Comment:
Maybe move this comment alongside the constant list of excluded CRIs?

Reply:
I think, we can remove this comment. Bacause, CRI_TYPES_EXCLUDED_FOR_DOB_CORRELATION list name is clear.